src/components/IFrame.js

```diff
@@ -17,73 +28,155 @@ function getNewDotURL(url) {
     }
 
     if (pathname === 'inbox') {
-        return 'home';
+        return ROUTES.HOME_OLDDOT;
     }
 
     if (pathname === 'expenses') {
-        return `${params.viewMode === 'charts' ? 'insights' : 'expenses'}${paramString ? `/?param=${paramString}` : ''}`;
+        return `${params.viewMode === 'charts' ? ROUTES.INSIGHTS_OLDDOT : ROUTES.EXPENSES_OLDDOT}${paramString ? `/?param=${paramString}` : ''}`;
     }
 
     if (pathname === 'admin_policies') {
         const {section} = params;
-        return section === 'individual' ? 'individual_workspaces' : 'group_workspaces';
+        return section === 'individual' ? ROUTES.INDIVIDUALS_OLDDOT : ROUTES.GROUPS_OLDDOT;
     }
 
     if (pathname === 'policy') {
-        const workspaceID = params.policyID || '';
-        const section = urlObj.hash.slice(1) || 'overview';
+        const workspaceID = params.policyID ?? '';
+        const section = urlObj.hash.slice(1) ?? '';
+
+        let sectionName = section;
+        switch (section) {
+            case '':
+                sectionName = 'overview';
+                break;
+            case 'js_policyEditor_perDiem':
+                sectionName = 'per-diem';
+                break;
+            case 'exportFormats':
+                sectionName = 'export-formats';
+                break;
+            default:
+                sectionName = section;
+        }
 
-        return `workspace/${workspaceID}/${section}`;
+        return ROUTES.WORKSPACES_OLDDOT.getRoute(workspaceID, sectionName);
     }
 
     if (pathname === 'settings') {
         const {section} = params;
-        return `settings/${section}`;
+
+        let sectionName = '';
+        switch (section) {
+            case undefined:
+            case '':
+                sectionName = 'account';
+                break;
+            case 'expenserules':
+                sectionName = 'expense-rules';
+                break;
+            case 'creditcards':
+                sectionName = 'cards';
+                break;
+            default:
+                sectionName = section;
+        }
+
+        return `settings/${sectionName}`;
     }
 
-    if (pathname.includes('domain')) {
-        return pathname;
+    if (pathname === 'admin_domains') {
+        return ROUTES.DOMAINS_OLDDOT;
     }
 
-    return pathname;
-}
+    if (pathname.includes('domain')) {
+        const [, section] = pathname.split('_');
```

Comment:
I haven't seen this syntax before. What is the longer equivalent of `[, section]`?

Reply:
There are parts copied from a different PR @hayata-suenaga 

But if I am right that means we are ignoring the first element from destructured array.

Reply:
yep

Domain URLs look something like the one below. The URLs are always prefixed with `domain`. We're splitting the URL path into `['domain', 'companycards']` and using only the second element (a string that represents the section name).

<img width="300" alt="Screenshot 2023-10-19 at 9 39 34 AM" src="https://github.com/Expensify/App/assets/98560306/b96920e0-3a61-4c39-b52d-6e90d03ea93e">

admin_domain